pkg/cmd/util/downloadrequest/downloadrequest.go

```diff
@@ -72,12 +72,6 @@ func Stream(ctx context.Context, kbClient kbclient.Client, namespace, name strin
 			return
 		}
 
-		// TODO: once the minimum supported Kubernetes version is v1.9.0, remove the following check.
-		// See http://issue.k8s.io/51046 for details.
-		if updated.Name != created.Name {
-			return
-		}
-
```

Comment:
I'm not quite sure I understand the relationship between this check and the issue referenced. The issue seems to be about adding fieldSelector support for CRDs, but where is Velero's use of field selectors dependent on the above check?

Reply:
According to the pr https://github.com/vmware-tanzu/velero/pull/3004#discussion_r533720827,
```
Pretty sure we can actually remove these checks now. However, let's not in this PR :)
```
There is a need to clean up the code.

Reply:
This is a forgotten child issue.

Reply:
I saw that. But I'm still unclear how these things are related. I don't doubt that this is likely unnecessary now, but I'm just not seeing how the change referenced there relates to whether or not we need this. How is this check related to the field selectors changed referenced in http://issue.k8s.io/51046